src/content/refi/index.md

```diff
@@ -0,0 +1,85 @@
+| **Opaque**: Process often handled via RFPs to source carbon based on type, volume, price, etc. Majority of trades take place via over-the-counter (OTC) deals negotiated with little-to-no record afterward. 				   							 | **Transparent**: All products and trades are immutably recorded and accessible to all interested parties in the DCM. Multiple offsetting pool types exist with defined criteria, pricing, and immediate liquidity.							  	  |
+| 		 		  																																		    							 |																																			          						  	  |
+| **High cost**: The process is time-consuming, with high engagement costs. There are numerous decentralized exchanges providing reliable trading outlets 24/7, reducing both time and fee costs. Broker and other fees, including margins, can range between 2.5% and 200%.	 | **Dynamic cost**: Costs are dynamically shown in real time. Fees are in the region of 0.5%.																				  	     						 	  |
+|      																		  																			   							 |																																				   						  	  |
+| **Not scalable**: Multiple product sources and intermediators make for a highly fragmented space. High friction due to due-diligence complications affecting market participants in one-off deals. High levels of counterparty risk.  							 | **Readily scalable**: Carbon bridging infrastructure leveraging blockchain technology provides immense scaling potential to build digital carbon credit liquidity. New automated infrastructure provides a highly efficient route to the blockchain-enabled DCM. |
+| 							   										  																												 |																																									        	  |
+| **No yield**: Holders of carbon credits lack additional return opportunity without trading and incurring additional costs.																	  								 | **High yield**: Through digital carbon assets, additional returns are possible by utilizing other DeFi-native applications that integrate carbon as collateral.															    	  |
+| 																																													 |																																										 	  |
+| **Slow speed**: Offsets are executed using a compulsory Voluntary Emission Reduction Purchase Agreement (VERPA) and typically take weeks to execute, and even longer if you are buying at scale.			  										 | **High speed**: Purchase any amount of carbon in minutes rather than weeks, at almost any size and with low slippage.																						  	  |

```

Comment:
```suggestion
| **Slow speed**: Offsets are executed using a compulsory **Voluntary Emission Reduction Purchase Agreement (VERPA)** and typically take weeks to execute, and even longer if you are buying at scale.			  										 | **High speed**: Purchase any amount of carbon in minutes rather than weeks, at almost any size and with low slippage.																						  	  |
```